doc/installation_instructions/platform/generic.rst

```diff
@@ -0,0 +1,51 @@
+Customizing environment variables
+=================================
+
+Various environment variables can be used to control the compilers and
+dependencies for SmartSim. These are particularly important to set before the
+``smart build`` step to ensure that the Orchestrator and machine-learning
+backends are compiled with the desired compilation environment.
+
+.. note::
```

Comment:
<img width="285" alt="Screen Shot 2023-02-28 at 10 11 53 AM" src="https://user-images.githubusercontent.com/30413257/221911572-2889cf9b-f319-41fb-9d0e-2ec32cc461da.png">

This is what the note format looks like on my end

Reply:
negative